lib/output.js

```diff
@@ -67,6 +71,14 @@ module.exports = function (parameters, response) {
     threshold
   ));
 
+  if (parameters.or) {
+    console.log('\nHere are your optimized images: ', optimizedResoruceURL);
+  }
+
+  if (parameters.download) {
+    got.stream(optimizedResoruceURL).pipe(createWriteStream('optimized.zip'));
+  }
```

Comment:
Yes, I do get the `async` part of it, but any other way here to pipe the stream, I could stick to `got.get(url, [options], [callback])`